.agent/workflows/lint.md

```
@@ -0,0 +1,180 @@
+}
+
+// Better (often):
+val out = xs.filter { it.isValid() }.map { Foo(it) }
+```
+
+2. Prefer `runCatching` when you are intentionally producing a value on failure:
+
+```kotlin
+// Smell: try/catch used only to default a value
+val bytes = try {
+	Files.readAllBytes(path)
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)